README.md

```diff
@@ -1 +1,34 @@
 # Oh Crop! Back End
+##Get Endpoints:
+__Motivational message__
+```
+oh-crop-be.herokuapp.com/api/v1/
+```
+
+__All Plants__
+```
+oh-crop-be.herokuapp.com/api/v1/plants
+```
+
+__Meet a random plant__
+```
+oh-crop-be.herokuapp.com/api/v1/plants/meet
+```
+
+__Get plant by ID__
+```
+oh-crop-be.herokuapp.com/api/v1/plants/2
+```
+
+Note: add a plant ID to the end... in the above example, plant ID = 2
+
+__Plant Search__
+```
+oh-crop-be.herokuapp.com/api/v1/plants/search?q=tomato
```

Comment:
This looks great, we should add some description on these as to what each endpoint does